pages/scholarship.vue

```diff
@@ -0,0 +1,92 @@
+          <h2 class="text-2xl font-light">Sponsored by Logicly Pty Ltd</h2>
+        </div>
+      </div>
+    </page-section>
+
+    <page-section>
+      <div class="px-0 xl:px-6 scholarship-link">
+        <div class="text-logiclytextgrey">
+          <h3 class="pb-2 text-lg">Logicly is offering up to eight individual scholarships for computer science graduates to complete a five-day <i>Introduction to Perl for Programmers</i> course, run by Paul Fenwick and valued at over $2,500 per person.</h3>
+          <h3 class="pb-6 text-lg">The scholarships are a fantastic opportunity for the successful applicants to receive a world-class introduction to the popular Perl language and to obtain the background and technical skills necessary to begin developing code in a professional setting.</h3>
+
+          <h4 class="pb-1 text-lg font-bold">Course participants will also be considered for a Graduate Programmer position at Logicly</h4>
+          <p class="pb-4 font-light">Applicants will ideally be recent computer science graduates or computer science students who are soon to complete their degree.</p>
+
+          <h4 class="pb-1 text-lg font-bold">What is Introduction to Perl for Programmers?</h4>
+          <p class="pb-2 font-light">Introduction to Perl for Programmers is an extended Perl workshop for those with a solid programming background but no specific or formal training in Perl. It familiarises students with the basic structure and design of the language and explores many of the features that have made Perl such a widely used and widely useful development platform.</p>
+          <p class="pb-2 font-light">The course will be run by Paul Fenwick, one of the world’s foremost Perl trainers.</p>
+          <p class="pb-4 font-light">Scholarship recipients will attend the five-day course to be run at Logicly’s office in Fitzroy from Monday 1st July to Friday 5th July, 2024.</p>
+
+          <h4 class="pb-1 text-lg font-bold">Who is Logicly?</h4>
+          <p class="pb-2 font-light">Logicly is a Melbourne based data-focused software development business providing services to Government, universities and corporates. With a heavy emphasis on Perl and open-source software, we specialise in user-focused on-line data collection and management projects. We work closely with our clients to develop user-friendly, secure, and considered digital information management solutions in areas such as mental health, education and transport.</p>
+          <p class="pb-2 font-light">Logicly is a collaborative, innovative organisation that values learning and growth.</p>
+          <p class="pb-4 font-light">Incorporated in 1996, we also trade under the name WebSurvey and were previously named “Strategic Data”. For more information about us, see <NuxtLink to="/" class="hover:underline" exact>https://www.logicly.com.au</NuxtLink> and <a href="http://www.websurvey.com.au">http://www.websurvey.com.au</a></p>
```

Comment:
I don't know if I'm doing this right, we'll find out! Can you please make ` WebSurvey` in `we also trade under the name WebSurvey` a link to the WebSurvey website, and delete the entire 'For more information about us...' sentence?